### ISSUE TITLE
updating R packages broke R serve connection

### ISSUE THREAD

**Issue by Rom1Garnier** [twosigma/beakerx#4268]
I was running Beaker just fine earlier today, then decided to update my packages from within Rbase. 

Since then, impossible to run R in Beaker. 

```
Jun 15, 2016 7:03:25 PM com.twosigma.beaker.r.utils.RServerEvaluator$workerThread startRserve
SEVERE: exception starting RServe
org.rosuda.REngine.Rserve.RserveException: eval failed, request status: error code: 127
    at org.rosuda.REngine.Rserve.RConnection.eval(RConnection.java:234)
    at com.twosigma.beaker.r.utils.RServerEvaluator$workerThread.startRserve(RServerEvaluator.java:444)
    at com.twosigma.beaker.r.utils.RServerEvaluator$workerThread.run(RServerEvaluator.java:473)
```

I have uninstalled both R and Beaker, and reinstalled all dependencies as well as the packages I normally use. Still not working.
I have checked the previous Beaker install for path preferences: none were set, so this shouldn't be the issue here. 

Any thoughts?

**Comment by scottdraves** [comment]
did you `install.packages('roxygen2')`?

**Comment by Rom1Garnier** [comment]
Yes, along with the other mentionned in the "Getting Started". 
I have tried re-running the command for roxygen2, to no avail.

**Comment by scottdraves** [comment]
confirmed, i reproduced this.

**Comment by mnavasiolava** [comment]
@scottdraves could you please check if RCurl package has been installed successfully (in terminal run `require('RCurl')`)

**Comment by Rom1Garnier** [comment]
@mnavasiolava I checked on mine. 'RCurl' wasn't properly installed, even after running `install.packages('Rcurl')`. 
However, after correcting that, the problem persists - exact same message as above.

**Comment by mnavasiolava** [comment]
@Rom1Garnier could you please check the other packages from the 'Getting started' page

```
require('Rserve');
require('ggplot2');
require('devtools');
require('RCurl');
require('RJSONIO');
require('jpeg');
require('png');
require('base64enc');
require('roxygen2');

```

**Comment by Rom1Garnier** [comment]
Fixed it. 
Somehow, `roxygen2` (which I re-installed yesterday) was not working properly. 

It was appearing in the list of packages within R base, was present in the library, but R wasn't able to load it. Trying to overwrite the install returned a message stating that R wasn't able to remove the previous installation. 

I had to shut down R, force shut a few Rscript.exe processes that lingered, and remove `roxygen2` from the R library manually. Then a clean`install.packages('roxygen2')`seems to have done the trick as I can properly access R from Beaker.  

I suspect the fact the package failed during the update process somehow?
